elastic/kibana
# [Infra UI] Only show 'Hosts' tour when in 'hosts' view

Closes #155590
## Summary

This PR shows hosts tour only when "hosts" is selected in the `Show` menu

## Testing

- Go to Inventory page
- Select hosts (default if not changed)
   - You should see the hosts tour
 
<img width="2223" alt="image" src="https://user-images.githubusercontent.com/14139027/234004293-4720cbed-a120-467a-beb5-1c57ed1cc87e.png">

- Select anything else (for example Kubernetes Pods, Docker Containers)
   - You shouldn't see  the hosts tour
  
<img width="2232" alt="image" src="https://user-images.githubusercontent.com/14139027/234004468-d5cac8e6-42dc-4d37-954d-18b32cdd0552.png">
